src/components/head-metadata/index.tsx

```diff
@@ -55,9 +55,8 @@ export default function HeadMetadata(props: HeadMetadataProps) {
 		productName ?? 'HashiCorp'
 	)
 
-	const ogImageUrl = `${getDeployedUrl(props.host)}/og-image/${
-		productSlug ?? 'base'
-	}.jpg`
+	const ogImagePath = props.localOgImage || `${productSlug ?? 'base'}.jpg`
```

Comment:
Adds support for a `props.localOgImage` path, which felt necessary to support a custom image for certifications (rather than the default `base.jpg`).